File: rafael/operadores/UseTernarioNegacaoIncrementoDecremento.cs
class UseTernarioNegacaoIncrementoDecremento 
{
	static void Main() 
	{	
		int a = 10;
		int b = 8;

		System.Console.WriteLine((a < b) ? a : b);
		System.Console.WriteLine(!(a < b) ? "Marcelo" : "Jonas");
		System.Console.WriteLine((a < b) ? a : ++b);
		System.Console.WriteLine(!(--a == b) ? a : b + 1);
	}
}

File: rafael/operadores/UseTernarioNegacaoIncrementoDecremento.java
class UseTernarioNegacaoIncrementoDecremento {
	public static void main(String[] args) {	
		int a = 10;
		int b = 8;

		System.out.println((a < b) ? a : b);
		System.out.println(!(a < b) ? "Marcelo" : "Jonas");
		System.out.println((a < b) ? a : ++b);
		System.out.println(!(--a == b) ? a : b + 1);
	}
}
